### ISSUE TITLE
[Database Update]: Add entries for a large set of missing id's from Redump.org

### ISSUE THREAD

**Issue by israpps** [israpps/HDL-Batch-installer#26]
### Game title

a lot

### Game ID

Script used to parse redump.org database:
 ```cmd
 @echo off
type PS2.DB |findstr "<rom"| findstr /V ".cue" | busybox.exe sed "s/ sha1=/;/g" | busybox.exe sed "s/ md5=/;/g" | busybox.exe sed "s/ size=/;/g" | busybox.exe sed "s/ crc=/;/g" | busybox.exe sed "s/<rom name=//g" | busybox.exe sed "s/^[[:space:]]*//" | busybox.exe sed "s/\/>//" >ndea

REM https://redump.org/discs/quicksearch/%d/
del LST.TXT
for /f "tokens=1,2,3,4,5 delims=;" %%a in (ndea) do (
wget -q --show-progress redump.org/discs/quicksearch/%%d/ -O - | findstr "Serial</th>" | busybox.exe grep -m 1 -o "[A-Z][A-Z][A-Z][A-Z]-[0-9][0-9][0-9][0-9][0-9]" >> LST.TXT
REM | busybox.exe tr "\n" "," | busybox.exe sed "$s/,$/\n/"
)

REM After running this script, the file was alphabetically ordered and cleaned (removed duplicate entries) by notepad++
 ``` 
 
```
KOEI_000.81
KOEI_652.47
PAPX_905.04
PAPX_905.14
PAPX_905.15
PAPX_905.16
PAPX_905.20
PAPX_905.23
PCPX_966.05
PCPX_966.09
PCPX_966.14
PCPX_966.15
PCPX_966.17
PCPX_966.18
PCPX_966.20
PCPX_966.21
PCPX_966.27
PCPX_966.31
PCPX_966.36
PCPX_966.38
PCPX_966.57
PCPX_980.17
PDSS_000.70
PDSS_023.18
SCED_503.81
SCED_505.20
SCED_506.10
SCED_506.15
SCED_507.33
SCED_507.80
SCED_509.89
SCED_509.90
SCED_511.66
SCED_512.69
SCED_512.87
SCED_513.52
SCED_515.63
SCED_515.96
SCED_515.98
SCED_516.44
SCED_516.57
SCED_517.79
SCED_519.05
SCED_519.42
SCED_520.53
SCED_520.54
SCED_521.37
SCED_521.47
SCED_521.77
SCED_522.70
SCED_522.71
SCED_523.11
SCED_524.36
SCED_524.52
SCED_524.55
SCED_525.78
SCED_528.41
SCED_528.47
SCED_528.70
SCED_529.35
SCED_530.18
SCED_531.77
SCED_532.98
SCED_533.16
SCED_533.25
SCED_533.49
SCED_535.22
SCED_536.84
SCED_537.33
SCED_538.54
SCED_539.16
SCED_539.32
SCED_539.39
SCED_539.62
SCED_540.36
SCED_541.92
SCED_542.38
SCED_543.63
SCED_544.81
SCED_544.95
SCED_545.23
SCED_545.58
SCED_545.67
SCED_546.03
SCED_546.92
SCES_510.03
SCES_517.25
SCES_521.37
SCES_544.98
SCES_545.03
SCES_545.04
SCES_545.56
SCES_547.65
SCES_547.69
SCES_548.52
SCES_548.54
SCES_548.56
SCES_550.61
SCES_550.96
SCES_550.97
SCES_551.07
SCES_551.27
SCES_551.28
SCES_551.77
SCES_552.57
SCES_552.59
SCES_552.62
SCES_554.01
SCES_554.21
SCES_554.25
SCES_554.39
SCES_554.85
SCES_555.13
SCES_555.21
SCES_555.51
SCES_555.53
SCES_556.12
SCES_556.16
SCES_556.19
SCES_556.50
SCKA_200.67
SCKA_201.40
SCPH_100.90
SCPH_701.20
SCPM_853.01
SCPM_853.02
SCPM_853.03
SCUS_280.28
SCUS_946.77
SCUS_971.78
SCUS_976.00
SCUS_978.99
SLAJ_250.15
SLAJ_250.17
SLAJ_250.28
SLAJ_250.41
SLAJ_350.07
SLED_502.86
SLED_504.39
SLED_504.84
SLED_504.89
SLED_505.87
SLED_508.95
SLED_510.12
SLED_510.47
SLED_510.48
SLED_510.62
SLED_510.74
SLED_510.91
SLED_511.27
SLED_511.86
SLED_512.12
SLED_512.25
SLED_512.64
SLED_513.38
SLED_513.64
SLED_513.80
SLED_516.51
SLED_516.76
SLED_519.02
SLED_519.21
SLED_519.75
SLED_519.87
SLED_519.92
SLED_519.93
SLED_520.14
SLED_523.04
SLED_523.26
SLED_524.85
SLED_525.06
SLED_525.97
SLED_527.36
SLED_528.73
SLED_528.75
SLED_528.79
SLED_528.90
SLED_528.91
SLED_529.79
SLED_530.66
SLED_530.97
SLED_531.37
SLED_531.98
SLED_533.30
SLED_533.65
SLED_534.40
SLED_534.42
SLED_534.45
SLED_535.37
SLED_535.91
SLED_536.19
SLED_536.25
SLED_536.37
SLED_536.50
SLED_536.64
SLED_536.81
SLED_537.11
SLED_537.70
SLED_538.88
SLED_539.54
SLED_539.77
SLED_539.83
SLED_540.22
SLED_543.15
SLED_543.27
SLED_543.28
SLED_544.01
SLED_544.45
SLED_545.09
SLED_550.22
SLES_501.21
SLES_504.20
SLES_508.55
SLES_509.01
SLES_516.45
SLES_516.47
SLES_518.27
SLES_519.61
SLES_523.92
SLES_524.47
SLES_529.07
SLES_529.34
SLES_533.96
SLES_534.20
SLES_534.65
SLES_536.34
SLES_537.93
SLES_538.05
SLES_544.33
SLES_544.34
SLES_544.35
SLES_544.36
SLES_544.47
SLES_544.72
SLES_545.32
SLES_545.36
SLES_546.12
SLES_546.94
SLES_547.12
SLES_547.37
SLES_547.44
SLES_547.46
SLES_549.00
SLES_549.23
SLES_549.38
SLES_549.40
SLES_549.85
SLES_549.87
SLES_550.23
SLES_551.66
SLES_551.72
SLES_551.96
SLES_552.01
SLES_552.64
SLES_552.65
SLES_553.38
SLES_553.43
SLES_553.66
SLES_554.11
SLES_554.33
SLES_554.34
SLES_555.02
SLES_555.26
SLES_556.04
SLKA_250.88
SLKA_251.68
SLKA_253.25
SLKA_253.85
SLKA_254.14
SLPM_252.62
SLPM_550.16
SLPM_551.58
SLPM_552.91
SLPM_610.01
SLPM_610.04
SLPM_610.07
SLPM_610.08
SLPM_610.09
SLPM_610.11
SLPM_610.13
SLPM_610.19
SLPM_610.20
SLPM_610.25
SLPM_610.26
SLPM_610.27
SLPM_610.29
SLPM_610.30
SLPM_610.35
SLPM_610.37
SLPM_610.39
SLPM_610.41
SLPM_610.42
SLPM_610.50
SLPM_610.52
SLPM_610.53
SLPM_610.54
SLPM_610.55
SLPM_610.59
SLPM_610.67
SLPM_610.72
SLPM_610.73
SLPM_610.74
SLPM_610.76
SLPM_610.77
SLPM_610.80
SLPM_610.84
SLPM_610.86
SLPM_610.87
SLPM_610.90
SLPM_610.91
SLPM_610.92
SLPM_610.93
SLPM_610.94
SLPM_610.96
SLPM_610.98
SLPM_611.00
SLPM_611.02
SLPM_611.03
SLPM_611.04
SLPM_611.05
SLPM_611.06
SLPM_611.09
SLPM_611.10
SLPM_611.12
SLPM_611.13
SLPM_611.14
SLPM_611.15
SLPM_611.16
SLPM_611.20
SLPM_611.21
SLPM_611.22
SLPM_611.23
SLPM_611.24
SLPM_611.25
SLPM_611.29
SLPM_611.30
SLPM_611.31
SLPM_611.32
SLPM_611.33
SLPM_611.34
SLPM_611.35
SLPM_611.37
SLPM_611.39
SLPM_611.40
SLPM_611.42
SLPM_611.44
SLPM_611.48
SLPM_611.50
SLPM_611.52
SLPM_611.54
SLPM_611.55
SLPM_611.57
SLPM_611.58
SLPM_611.61
SLPM_611.63
SLPM_650.31
SLPM_650.35
SLPM_651.62
SLPM_651.72
SLPM_652.01
SLPM_652.11
SLPM_654.36
SLPM_656.56
SLPM_666.99
SLPM_667.80
SLPM_668.34
SLPM_669.04
SLPM_669.25
SLPM_669.28
SLPM_675.25
SLPM_685.01
SLPM_685.02
SLPM_685.05
SLPM_685.09
SLPM_685.19
SLPM_685.21
SLPM_685.23
SLPS_256.52
SLPS_656.68
SLPS_999.99
SLUS_000.00
SLUS_123.45
SLUS_206.00
SLUS_215.11
SLUS_216.17
SLUS_219.24
SLUS_280.34
SLUS_280.52
SLUS_280.59
SLUS_280.67
SLUS_280.68
SLUS_280.69
SLUS_290.14
SLUS_290.85
SLUS_291.47
SLUS_291.51
TLES_820.43
 
```

### game MD5 Checksum

dsafafafeva

### other checks

- [ ] I am using the [Latest version](https://github.com/israpps/HDL-Batch-installer/releases/)

### Game source

fawvevaw

**Comment by israpps** [comment]
last script:
```cmd
@echo off
type PS2.DB |findstr "<rom"| findstr /V ".cue" |  busybox.exe tr ";" "," | busybox.exe sed "s/ sha1=/;/g" | busybox.exe sed "s/ md5=/;/g" | busybox.exe sed "s/ size=/;/g" | busybox.exe sed "s/ crc=/;/g" | busybox.exe sed "s/<rom name=//g" | busybox.exe sed "s/^[[:space:]]*//" | busybox.exe sed "s/\/>//" >ndea





REM https://redump.org/discs/quicksearch/%d/
del LST.TXT
del FULL_LIST.DB
for /f "tokens=1,2,3,4,5 delims=;" %%a in (ndea) do (
..\wget -q --show-progress redump.org/discs/quicksearch/%%d/ -O TMP
type TMP | findstr "Serial</th>" | busybox.exe grep -m 1 -o "[A-Z][A-Z][A-Z][A-Z][[:blank:]\-][0-9][0-9][0-9][0-9][0-9]" | busybox.exe tr "\n" "," | busybox.exe sed "$s/,$/;/" >>FULL_LIST.DB
type TMP | findstr /C:"<title>redump.org &bull;" | busybox sed "s/<title>redump.org &bull; //" |  busybox sed "s/<\/title>//" | busybox.exe tr "\n" "," |busybox.exe sed "$s/,$/;/" >>FULL_LIST.DB
echo %%d >>FULL_LIST.DB
REM | busybox.exe tr "\n" "," | busybox.exe sed "$s/,$/\n/"
)

```

**Comment by israpps** [comment]
merged in [fbbe9c6](https://github.com/israpps/HDL-Batch-installer/commit/fbbe9c6360f86f64bb4629b34019f628dc12896c)
